packages/components/src/components/switch/switch-styles.spec.ts

```diff
@@ -37,7 +37,7 @@ describe('getComponentCss()', () => {
     theme: Theme;
   }>([
     {
-      alignLabel: 'right',
+      alignLabel: 'end',
```

Comment:
Are `left` and `right` even covered now?

Reply:
yes, as vrt test